benjamin-firth/ican2_FE
# NavBar styling and routing

#### What's this PR do?
This PR adds styling to the NavBar, renders appropriate buttons for each page, and routes the user to the correct page.
#### Where should the reviewer start?
View the `App.js` file to see the props being sent to `<NavBar />`, then checkout `NavBar.js` and `NavButton.js`.
#### How should this be manually tested?
Ensure the buttons are rendering on the page in the browser and navigate you to the correct pages.
